runner.yml.dist

```diff
@@ -35,6 +35,7 @@ commands:
         $config['cas.settings']['server']['hostname'] = 'authentication';
         $config['cas.settings']['server']['port'] = '8001';
         $config['cas.settings']['server']['path'] = '/';
+        $config['cas.settings']['server']['version'] = '3';
```

Comment:
Please see above comment. If version 3 is needed for PCAS, this is the only place it needs to be, not in the .install hook.

Reply:
As discussed and based on the source of the cas module and pcas server lib, it needs to stay like this to let the module use the proper url argument (url <> service).